libs/services/license/src/lib/barcode.service.ts

```diff
@@ -4,13 +4,16 @@ import {
 } from '@island.is/clients/license-client'
 import { ConfigType } from '@nestjs/config'
 import { Inject, Injectable } from '@nestjs/common'
-import { Cache as CacheManager } from 'cache-manager'
+import { Cache as CacheManager, Milliseconds } from 'cache-manager'
 import { sign, VerifyOptions, verify } from 'jsonwebtoken'
 import { LICENSE_SERVICE_CACHE_MANAGER_PROVIDER } from './licenseCache.provider'
 import { LicenseConfig } from './license.config'
 
 export const BARCODE_EXPIRE_TIME_IN_SEC = 60
+export const BARCODE_SESSION_EXPIRE_TIME_IN_SEC = 1800 // 30 minutes
```

Comment:
It's been requested to make these two constants configurable with environment variables.